### PR TITLE
 Refactored tests and permissions checking to handle edge cases for PermissibleRender

### PR DESCRIPTION
**Pull request by kjendrzyca**
I want to use this package so I decided to fix the tests first.

- cleaned up and fixed the tests in `permissibleRender.test`
- removed global variable for props in `permissibleRender.test` (it was replaced in almost every test)
- handled edge case where required props are not specified
- added tests for checking what happens when `requiredPermissions` and `userPermissions` are both empty

### CODE REVIEW

File: test/permissibleRender.test.js [hunk 1]
```diff
@@ -152,12 +170,13 @@ describe('PermissibleRender', () => {
     searchedElement.length.should.be.greaterThan(0);
   });
 
-  it('doesn\'t render a component if the user hasn\'t all of necessary permissions when `oneperm` prop isn\'t defined', () => {
-    props = {
+  it('doesn\'t render a component if the user doesn\'t have all of necessary permissions when `oneperm` prop is explicitely set to false', () => {
```
Comment (damian-brainhub) [inline test/permissibleRender.test.js]: Typo. Please use `explicitly` instead of `explicitely`

Comment (kjendrzyca) [reply test/permissibleRender.test.js]: Fixed.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@brainhubeu/react-permissible",
-  "version": "1.0.2",
+  "version": "1.1.0",
```
Comment (adam-golab) [inline package.json]: :+1:
